src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +21,13 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .flatMap(element -> Arrays.stream(element.split(COMMA)))
+                .map(Integer::valueOf)
+                .filter(number -> number % DIVISOR_FOR_EVEN_CHECK == 0)
+                .min(Integer::compare)
```

Comment:
I don't think it's wrong, but I think it's better.
```suggestion
                .mapToInt(Integer::valueOf)
                .filter(number -> number % DIVISOR_FOR_EVEN_CHECK == 0)
                .min()
```